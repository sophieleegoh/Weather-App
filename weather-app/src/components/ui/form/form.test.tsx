import renderer from "react-test-renderer";
import Form from "./index";

describe("Form snapshot tests", () => {
  it("Renders the component", () => {
    const component = renderer.create(
      <Form>
        <div>This is a form</div>
      </Form>
    );

    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });
});
