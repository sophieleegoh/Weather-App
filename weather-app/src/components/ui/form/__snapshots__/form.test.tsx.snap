// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form snapshot tests Renders the component 1`] = `
<form
  onSubmit={[Function]}
>
  <div
    className="grid grid-cols-1 md:grid-cols-2 gap-4"
  >
    <div>
      This is a form
    </div>
  </div>
</form>
`;
